cmd/filestransfer/filestransfer.go

```diff
@@ -0,0 +1,180 @@
+// Example: Files transfer using PUT/GET commands
+//
+// This example shows how to transfer files to staging area, from which data can be loaded into snowflake
+// database tables. Apart from sending files to staging area using PUT command, files can also be downloaded
+// using GET command.
+package main
+
+import (
+	"bytes"
+	"compress/gzip"
+	"database/sql"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	sf "github.com/snowflakedb/gosnowflake"
+)
+
+const customFormatCsvDataToUpload = "NUM; TEXT\n1; foo\n2; bar\n3; baz"
+
+func getDSN() (string, *sf.Config, error) {
+	env := func(key string, failOnMissing bool) string {
+		if value := os.Getenv(key); value != "" {
+			return value
+		}
+		if failOnMissing {
+			log.Fatalf("%v environment variable not set", key)
+		}
+		return ""
+	}
+	account := env("SNOWFLAKE_TEST_ACCOUNT", true)
+	user := env("SNOWFLAKE_TEST_USER", true)
+	password := env("SNOWFLAKE_TEST_PASSWORD", true)
+	database := env("SNOWFLAKE_TEST_DATABASE", true)
+	warehouse := env("SNOWFLAKE_TEST_WAREHOUSE", true)
+	host := env("SNOWFLAKE_TEST_HOST", false)
```

Comment:
Can you connect this property to the rest of "url properties" (port, protocol)?

Reply:
Done

Reply:
done